tests/ieee802154_hal/main.c

```diff
@@ -229,6 +229,10 @@ static int _init(void)
 
     ieee802154_radio_config_phy(ieee802154_hal_test_get_dev(RADIO_DEFAULT_ID), &conf);
 
+    /* ieee802154_radio_set_cca_mode*/
+    ieee802154_radio_set_cca_mode(ieee802154_hal_test_get_dev(RADIO_DEFAULT_ID), IEEE802154_CCA_MODE_ED_THRESHOLD);
+    ieee802154_radio_set_cca_threshold(ieee802154_hal_test_get_dev(RADIO_DEFAULT_ID), -35);
```

Comment:
there's now `CONFIG_IEEE802154_CCA_THRESH_DEFAULT`. Maybe you could reuse that one here

Reply:
Do they yield the same value?

Reply:
nope, but we could update it. Or this radio could redefine it

Reply:
I used the default!

Reply:
ah yes, it changed with #15237

Reply:
note that #15237 also added the `set_cca_threshold` part (but not the `set_cca_mode`). Just to be sure that the rebase doesn't add a duplicated `set_cca_threshold`

Reply:
Nice catch!